pkg/cmd/server/origin/controller.go

```diff
@@ -86,6 +86,11 @@ func (c *MasterConfig) NewKubernetesControllerInitalizers(kc *kubernetes.MasterC
 		ret[name] = initFn
 	}
 
+	// NB: this is funky -- it's actually a Kubernetes controller, but we run it as an OpenShift controller in order
+	// to get a handle on OpenShift clients, so that our delegating scales getter can work.  Ergo,
+	// we need to just not do anything right now.
+	delete(ret, "horizontalpodautoscaling")
```

Comment:
was this necessary? looking at how these are called, openshift ones with the same name take priority. 

I could see adding a check here to make sure "horizontalpodautoscaling" is registered so our override is actually overriding something